cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -46,24 +46,22 @@
 namespace modmesh
 {
 
-class RScene
-    : public Qt3DCore::QEntity
+class RScene : public Qt3DCore::QEntity
 {
-
 public:
 
-    explicit RScene(Qt3DCore::QNode * parent = nullptr)
-        : Qt3DCore::QEntity(parent)
-        , m_controller(new ROrbitCameraController(this))
+    explicit RScene(QNode * parent = nullptr)
+        : QEntity(parent)
     {
+        m_controller = new ROrbitCameraController(this);
     }
 
-    Qt3DExtras::QAbstractCameraController const * controller() const { return m_controller; }
-    Qt3DExtras::QAbstractCameraController * controller() { return m_controller; }
+    CameraController * controller() const { return m_controller; }
+    Qt3DExtras::QAbstractCameraController * qtController() const { return m_controller->asQtCameraController(); }
```

Comment:
Why do we need an helper `qtController()` for casting?  Is there a reason that the consumers cannot do it themselves?

```cpp
Qt3DExtras::QAbstractCameraController * qtc = scene->controller()->asQtCameraController();
```

If you want to avoid `Qt3DExtras::QAbstractCameraController const *`, an overload of `CameraController * controller()` should work.

Reply:
I'll make `CameraController` inherit `QAbstractCameraController`, so `qtController` will be removed